test/common/testinfra.py

```diff
@@ -46,7 +46,7 @@
 WHITELIST_LOCAL = "~/.config/github-whitelist"
 
 HOSTNAME = socket.gethostname().split(".")[0]
-DEFAULT_IMAGE = os.environ.get("TEST_OS", "fedora-25")
+DEFAULT_IMAGE = os.environ.get("TEST_OS", "fedora-26")
```

Comment:
Maybe we shouldn't make this the default image yet. After it's in at least beta would probably be a better time.